thorin/check.h

```diff
@@ -23,13 +23,15 @@ class Infer : public Def {
     Infer* unset() { return Def::unset()->as<Infer>(); }
     ///@}
 
-    /// @name union-find
-    ///@{
+    /// Eliminate Infer%s that have may have been resolved in the meantime by rebuilding.
```

Comment:
"Eliminate Infer%s that ~~have~~ may have been resolved in the meantime by rebuilding"